### DISCUSSION TITLE
Fui tentar criar uma Infrastructure Layer para o domínio Rico , porém EF Core dá erro

### DISCUSSION THREAD

**Discussion by SimpleDevSam** [balta-io/forum#630] [Tira Dúvidas] [answered]
Bom dia, mestre Balta!

Estou usando oq aprendi no seu curso pra criar um web server.

Criei o domínio rico.
Criei a application layer
E agora criei uma infrstructure layer e configurei as entidades, porém esta dando esse erro:

Unable to create a 'DbContext' of type ''. The exception 'The expression 'c => c.Address.Notifications' is not a valid member access expression. The expression should represent a simple property or field access: 't => t.MyProperty'. (Parameter 'memberAccessExpression')' was thrown while attempting to create an instance. For the different patterns supported at design time, see https://go.microsoft.com/fwlink/?linkid=851728

Estou enviando aqui pra saber se já viu algo parecido, ainda mais por estar mencionando a propriedade do Notifiable..

Pela minhas pesquisas, parece que precisaria de alguam forma ignorar essa propriedade, mas não estou conseguindo achar solução.

Link do repositório https://github.com/SimpleDevSam/RepublicsApp

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @SimpleDevSam, como vai?

Posta por favor as duas classes e o mapeamento de ambas aqui para podermos analisar melhor.
Pelo que me parece, está tentando informar uma classe ou lista onde é para informar uma propriedade.

Mas coloca o código aí com os detalhes, explica certinho o que você está tentando fazer, qual o cenário!

**Comment by KIQxl** [comment]
Boa tarde @SimpleDevSam, tudo bem?
eu enfrentei esse mesmo problema a algum tempo atrás. A solução foi, dentro da classe de contexto, no método OnModelCreating onde informamos ao Entity o mapeamento das entidades que temos, adicionar uma linha com o seguinte código:
"modelBuilder.Ignore\<Notification>\();"

isso faz o Entity ignorar as propriedades que vem da classe Notifiable, que é usada nas domain notifications do Flunt.

**Reply by SimpleDevSam** [reply]
Obrigado amigo! Tinha conseguido desta mesma forma, esqueci de vir aqui comentar.

**Comment by SimpleDevSam** [comment]
Boa tarde, amigos!

Acabou que estava logado no github do trabalho e não vi as atualizações aqui, nem vim concluir a resolução que achei.

A resolução que tive foi a mesma que o KIQxl mandou. Utilizar o modelBuilder.Ignore<Notification>(); no OnModelCreating.

Obrigado pessoal! E foi mal pela demora.
